Ginger/GingerCoreNET/ActionsLib/General/ActSetVariableValue.cs

```diff
@@ -274,7 +274,11 @@ public override void Execute()
                 Error = "Unknown set " + GingerDicser.GetTermResValue(eTermResKey.Variable) + " value operation.";
                 return;
             }
-
+            if (Var.Value.Contains("Value is at the last in the list"))
+            {
+                Status = Amdocs.Ginger.CoreNET.Execution.eRunStatus.Failed;
```

Comment:
1- change the GetAuto method to be "GenerateAutoValue(ref string errorMsg)"
2- use the errorMsg to know in the Act if auto value set failed and why